staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/flunder/etcd.go

```diff
@@ -44,3 +47,28 @@ func NewREST(scheme *runtime.Scheme, optsGetter generic.RESTOptionsGetter) (*reg
 	}
 	return &registry.REST{store}, nil
 }
+
+// NewStatusREST makes a RESTStorage for status that has more limited options.
+// It is based on the original REST so that we can share the same underlying store
+func NewStatusREST(scheme *runtime.Scheme, rest *registry.REST) *StatusREST {
+	statusStore := *rest.Store
+	statusStore.CreateStrategy = nil
+	statusStore.DeleteStrategy = nil
+	statusStore.UpdateStrategy = NewFlunderStatusStrategy(scheme)
```

Comment:
The sample needs heavier commenting.  Explain why you've deleted some and overwritten others to limit to just update with different rules

Reply:
updated. ptal, thanks.